src/kernel/vmm.zig

```diff
@@ -305,7 +305,10 @@ pub fn VirtualMemoryManager(comptime Payload: type) type {
         /// Error: pmm.PmmError
         ///     Bitmap(u32).Error.OutOfBounds - The address given is outside of the memory managed
         ///
-        pub fn isSet(self: *const Self, virt: usize) bitmap.Bitmap(u32).BitmapError!bool {
+        pub fn isSet(self: *const Self, virt: usize) bitmap.BitmapError!bool {
+            if (virt < self.start) {
+                return bitmap.BitmapError.OutOfBounds;
+            }
```

Comment:
Do the exiting tests cover this new validation, or should there be an additional tests

Reply:
It isn't covered, thanks for spotting that.